### PR TITLE
Fixes ENS suggestion on settings input username.

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Fixes ENS suggestion on settings input username.

fixes https://github.com/EveripediaNetwork/issues/issues/810

### CODE REVIEW

File: src/components/Settings/ProfileSettings.tsx [hunk 1]
```diff
@@ -206,7 +206,23 @@ const ProfileSettings = ({ settingsData }: ProfileSettingsProps) => {
         <VStack flex="2" align="left" spacing={4}>
           {/* PROFILE: USER NAME */}
           <FormControl isRequired isInvalid={inputUsername.error !== ''}>
-            <FormLabel htmlFor="username">Username</FormLabel>
+            <Flex>
+              <FormLabel htmlFor="username">Username</FormLabel>
+              {userENSAddr && (
+                <Text
```
Comment (Royal-lobster) [inline src/components/Settings/ProfileSettings.tsx:212]: @Adebesin-Cell Make it a button with variant `unstyled` or `link` so we will get keyboard accessibility and pointer on hover.

File: src/components/Settings/ProfileSettings.tsx [hunk 2]
```diff
@@ -206,7 +206,23 @@ const ProfileSettings = ({ settingsData }: ProfileSettingsProps) => {
...
+                  fontSize="xs"
+                  mb="2"
+                  onClick={() => {
+                    setInputUsername({
+                      value: userENSAddr,
+                      error: validateUsername(userENSAddr),
+                    })
+                  }}
+                >
+                  (Click to use your ENS {userENSAddr})
```
Comment (Royal-lobster) [inline src/components/Settings/ProfileSettings.tsx:222]: i think we can remove the message or show some other message if the username is already the user's ENS.

CleanShot 2022-10-17 at 19 32 39@2x

Comment (Adebesin-Cell) [reply src/components/Settings/ProfileSettings.tsx:222]: I think removing the message is fine.

Comment (Softdev1) [reply src/components/Settings/ProfileSettings.tsx:222]: I have the same comment as that of @Royal-lobster  once the changes is made, we are good to go

**Review by Royal-lobster — APPROVED** [review]

**Pull request merged**